kahuna/public/js/components/gr-chips/gr-chips.html

```diff
@@ -0,0 +1,18 @@
+<div class="gr-chips__wrapper"
+     ng:click="$grChipsCtrl.wrapperClicked($event)">
```

Comment:
See [previous response](https://github.com/guardian/grid/pull/1780#discussion_r53762472), you need different names if you want to be able to refer to different controllers.